entrypoint_scripts/os/linux.sh

```diff
@@ -97,6 +97,10 @@ function ubuntu_wkhtml_install() {
 	    wget https://downloads.wkhtmltopdf.org/0.12/0.12.5/wkhtmltox_0.12.5-1.xenial_amd64.deb
 	    dpkg -i wkhtmltox_0.12.5-1.xenial_amd64.deb 
 	    ;;
+        "19")
```

Comment:
As Ubuntu and Linux Mint are handled the same way here, this structure creates code duplication, making it harder for maintenance.
```suggestion
        "18.04" | "19")
```